src/backend/gog/games.ts

```diff
@@ -383,11 +384,17 @@ class GOGGame extends Game {
           ? wineExec.replaceAll("'", '')
           : wineExec
 
-      wineFlag.push(
-        ...(wineType === 'proton'
-          ? ['--no-wine', '--wrapper', `'${wineBin}' run`]
-          : ['--wine', wineBin])
-      )
+      if (wineType === 'proton') {
+        wineFlag.push(...['--no-wine', '--wrapper', `'${wineBin}' run`])
+      } else if (wineType === 'bottles') {
+        commandEnv = {
+          ...commandEnv,
+          HGL_BOTTLE_NAME: gameSettings.bottlesBottle
+        }
```

Comment:
Can't test right now, but wouldn't
```suggestion
        commandEnv.HGL_BOTTLE_NAME = gameSettings.bottlesBottle
```
work?